### PR TITLE
Fapi extend Fapi_Import

### PR DESCRIPTION
**Pull request by JuergenReppSIT**
* A new feature for importing keys based on binary blobs with Fapi_Import was added.
* Integration tests were extended to test this feature.
* A fix to Fapi_ChangeAuth was added.

### CODE REVIEW

File: Makefile-test.am [hunk 1]
```diff
@@ -1363,353 +1363,353 @@ test_integration_sapi_policy_authorizeNV_int_SOURCES = test/integration/main-sap
...
-test_integration_fapi_platform_certificates_fint_LDFLAGS = $(TESTS_LDFLAGS)
+test_integration_fapi_platform_certificates_fint_LDFLAGS = $(TESTS_LDFLAGS)  $(FAPI_LDFLAGS)
 test_integration_fapi_platform_certificates_fint_SOURCES = \
     test/integration/fapi-platform-certificates.int.c \
     test/integration/main-fapi.c test/integration/test-fapi.h
 
 test_integration_fapi_key_create_sign_fint_CFLAGS  = $(TESTS_CFLAGS)
-test_integration_fapi_key_create_sign_fint_LDADD   = $(TESTS_LDADD)
-test_integration_fapi_key_create_sign_fint_LDFLAGS = $(TESTS_LDFLAGS)
+test_integration_fapi_key_create_sign_fint_LDADD   = $(TESTS_LDADD)  -ljson-c
```
Comment (AndreasFuchsSIT) [inline Makefile-test.am:1379]: isn't this part of the FAPI_LDFLAGS in the following line ?

File: Makefile-test.am [hunk 2]
```diff
@@ -1363,353 +1363,353 @@ test_integration_sapi_policy_authorizeNV_int_SOURCES = test/integration/main-sap
 if FAPI
 test_integration_fapi_get_random_fint_CFLAGS  = $(TESTS_CFLAGS)
 test_integration_fapi_get_random_fint_LDADD   = $(TESTS_LDADD)
-test_integration_fapi_get_random_fint_LDFLAGS = $(TESTS_LDFLAGS)
+test_integration_fapi_get_random_fint_LDFLAGS = $(TESTS_LDFLAGS)  $(FAPI_LDFLAGS)
```
Comment (AndreasFuchsSIT) [inline Makefile-test.am:1366]: There are double-spaces.
Also in all following additions of FAPI_LDFLAGS.

File: src/tss2-fapi/api/Fapi_ChangeAuth.c [hunk 3]
```diff
@@ -156,6 +156,9 @@ Fapi_ChangeAuth_Async(
 
     TSS2_RC r;
 
+    /* Cleanup command context. */
+    memset(&context->cmd.Entity_ChangeAuth, 0, sizeof(IFAPI_Entity_ChangeAuth));
```
Comment (AndreasFuchsSIT) [inline src/tss2-fapi/api/Fapi_ChangeAuth.c:160]: Should this be backported to 2.4.1 ?

Comment (JuergenReppSIT) [reply src/tss2-fapi/api/Fapi_ChangeAuth.c:160]: yes, I think it should be backported.

**Pull request merged**